rules/REQUEST-932-APPLICATION-ATTACK-RCE.conf

```diff
@@ -649,7 +692,6 @@ SecRule FILES|REQUEST_HEADERS:X-Filename|REQUEST_HEADERS:X_Filename|REQUEST_HEAD
     setvar:'tx.rce_score=+%{tx.critical_anomaly_score}',\
     setvar:'tx.inbound_anomaly_score_pl1=+%{tx.critical_anomaly_score}'"
 
-
```

Comment:
This line should probably be put back in for consistency.

Reply:
Ack.